### ISSUE TITLE
Update the https://giraf.cs.aau.dk/ website

### ISSUE THREAD

**Issue by mathiasandresen** [aau-giraf/wiki#198]
**Is your feature request related to a problem? Please describe.**
The https://giraf.cs.aau.dk/ website is not updated to reflect changes in the project.

**Describe the solution you'd like**
To the website to be updated, or changed.
Maybe the websites source should be in a repository.

**Describe alternatives you've considered**
none

**Additional context**
From the website:
![billede](https://user-images.githubusercontent.com/5781825/80962404-487b9880-8e0d-11ea-9650-650ac0c2190e.png)

**Comment by mathiasandresen** [comment]
@Lortemanden I would like to argue, that this would be nice to have done before the end of the project, since this is what new student sees before starting the semester. It is also what the "public" sees.
Also Ulrik talked about updating this in the beginning of the semester.

**Comment by mathiasandresen** [comment]
the page has now been updated
